deploy/charts/ray/templates/raycluster.yaml

```diff
@@ -92,7 +92,7 @@ spec:
   # Note dashboard-host is set to 0.0.0.0 so that Kubernetes can port forward.
   headStartRayCommands:
     - ray stop
-    - ulimit -n 65536; ray start --head --no-monitor --dashboard-host 0.0.0.0
+    - ulimit -n 65536; ray start --head --port=6379 --no-monitor --dashboard-host 0.0.0.0
```

Comment:
Out of curiosity, why explicit `--port=6379` is needed now?

Reply:
Otherwise, if I understand correctly, a random-ish GCS server port will be assigned.
However, due to the missing port argument, the Ray operator, which runs the Monitor in a separate pod, will [assume](https://github.com/ray-project/ray/blob/d295a9d5451b40ed05f926aea2d0f534a1e2050b/python/ray/ray_operator/operator.py#L110) a default of 6379, which would cause an error in no-redis mode.

The rest of the example configs (for AWS VM autoscaling etc.) specify explicit ports, so this happens to make things more consistent.

Reply:
Or do we now run the GCS server port on 6379 by default?
In any case, safety and consistency is better!

Reply:
Yes, Ray does choose a consistent GCS port of 6379 in 1.11,0. Otherwise, legacy autoscaler configs and scripts with `ray start --adress=....:6379` will not longer work. Specifying `--port` on the head node sounds good!